TerrorConsole/Assets/_Main/Scripts/Objects/Switchs/SwitchCameraActivation.cs

```diff
@@ -0,0 +1,17 @@
+using UnityEngine;
+
+namespace TerrorConsole
+{
+    public class SwitchCameraActivation : CameraActivation, ISwitchInteractable
+    {
+        public void SwitchOn()
+        {
+            LeverActivated();
+        }
+
+        public void SwitchOff()
+        {
+            
+        }
+    }
+}
```

Comment:
Remove this class, DM me after solving every other comment so I can show you the magic of ´UnityAction´